modules/oe_content_featured_media_field/tests/src/FunctionalJavascript/FeaturedMediaEntityBrowserWidgetTest.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_featured_media_field\FunctionalJavascript;
+
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\field\Entity\FieldStorageConfig;
+
+/**
+ * Tests the output of "oe_featured_media_entity_browser" widget.
+ *
+ * @group oe_content_featured_media_field
+ */
+class FeaturedMediaEntityBrowserWidgetTest extends FeaturedMediaFieldWidgetTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'views',
+    'block',
+    'views_ui',
+    'system',
+    'oe_content_featured_media_test',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $view_display_options = [
+      'type' => 'oe_featured_media_label',
+      'label' => 'above',
+      'settings' => [
+        'link' => TRUE,
+      ],
+    ];
+
+    FieldStorageConfig::load('node.featured_media_field')
+      ->setCardinality(FieldStorageDefinitionInterface::CARDINALITY_UNLIMITED)
+      ->save();
+
+    /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $form_display */
+    $form_display = $this->container->get('entity_type.manager')
```

Comment:
Below you are accessing the service via "\Drupal::service('entity_display.repository')". Do we need to use container here?